test/javascript/components/dropdown.js

```diff
@@ -37,6 +44,54 @@ describe('Dropdown', function() {
       sinon.assert.called(spy);
       Foundation.Keyboard.trapFocus.restore();
     });
+
+    it('should open dropdown on button click', function() {
+      $dropdownController = $(getDropdownController()).appendTo('body');
+      $dropdownContainer = $(getDropdownContainer()).appendTo('body');
+      plugin = new Foundation.Dropdown($dropdownContainer, {});
+      plugin.open();
+
+      $dropdownController.on('show.zf.dropdown', function() {
+        $('#my-dropdown').should.not.be.hidden;
+        $('#my-dropdown').should.have.attr('aria-hidden', 'false');
+        $('#my-dropdown').should.have.class('is-open');
+        done();
+      });
+      plugin.close();
+    });
+
+    it('should open dropdown on hover', function() {
+      $dropdownController = $(getDropdownController()).appendTo('body');
+      $dropdownContainer = $(getHoverDropdownContainer()).appendTo('body');
+      plugin = new Foundation.Dropdown($dropdownContainer, {});
+      plugin.open();
+
+      $dropdownController.on('show.zf.dropdown', function() {
+        $('#my-dropdown').should.not.be.hidden;
+        $('#my-dropdown').should.have.attr('aria-hidden', 'false');
+        $('#my-dropdown').should.have.class('is-open');
+        done();
+      });
+      plugin.close();
+    });
```

Comment:
These functions do not test "click" and "hover" dropdowns but if the `.open()` works.

So you can delete one of them and replace with simply `should open dropdown`, OR provide a test that simulate the `hover` to test if it opens the dropdown (and close after).